sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/bitwiseExpressions.scala

```diff
@@ -27,8 +27,16 @@ import org.apache.spark.sql.types._
  * Code generation inherited from BinaryArithmetic.
  */
 @ExpressionDescription(
-  usage = "a _FUNC_ b - Bitwise AND.",
-  extended = "> SELECT 3 _FUNC_ 5; 1")
+  usage = "expr1 _FUNC_ expr2 - Bitwise AND.",
+  extended = """
+    Arguments:
+      expr1 - an integral numeric expression.
```

Comment:
Oh, you meant what is that. I referred `IntegralType` class. maybe I should fix that just to `intergral expression`.